discordbot.py

```diff
@@ -32,7 +32,17 @@ def embed_from_post(post):
 	embed_content = post.content[:2048]
 	embed_url = post.url
 	embed_timestamp = post.timestamp
-	embed_color = discord.Embed.Empty
+	
+	if board_name == "Ask":
+		embed_color = discord.Color.blue()
+	elif board_name == "Share":
+		embed_color = discord.Color.dark_teal()
+	elif board_name == "Tutorials":
+		embed_color = discord.Color.green()
+	elif board_name == "Announcements":
+		embed_color = discord.Color.dark_blue()	
+	else:
+		embed_color = discord.Embed.Empty
```

Comment:
Can you make these single quotes to make it more consistent with the rest of the code?

Reply:
Done!